jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2705,6 +2705,397 @@ YUI.add('juju-env-api', function(Y) {
     },
 
     /**
+      Return information about all entities in this model.
+
+      @param {Function} callback A callable that must be called once the
+        operation is performed. It will receive two arguments: an error string
+        and an empty object if an error occurred, null and the status result
+        otherwise. The status result is an object similar to the following:
+
+          {
```

Comment:
In theory, it should not change. It could be extended, but not modified. There is no command to generate that output, except, of course, the env.fullStatus itself. I think that documenting the value passed to callbacks is always valuable. Inline comments are not required in this case, as the logic is pretty much trivial.